### DISCUSSION TITLE
The return value of multi-version search

### DISCUSSION THREAD

**Discussion by momo91377** [web-infra-dev/rspress#1303] [Q&A] [answered]
背景：使用rspress搭建的文档内支持了多版本，但是在使用搜索功能时，搜索结果会返回多版本内相关内容，是否有什么配置可以实现搜索仅返回当前版本相关内容，或者在后续版本是否有考虑相关的支持？

**Comment by Timeless0911** [comment, marked as answer]
https://rspress.dev/guide/default-theme/multi-version#limit-search-to-current-version-only

**Reply by momo91377** [reply]
之前看文档没看到这里，感谢，我尝试一下👍🏻
